src/toil/test/wdl/builtinTest.py

```diff
@@ -58,6 +62,46 @@ def testFn_Floor(self):
         assert floor(1.999) == 1
         assert floor(-1.5) == -2
 
+    def testFn_ReadInt(self):
+        """Test the wdl built-in functional equivalent of 'read_int()'."""
+        num = 10
+        path = self._write_temp_file('read_int', content=str(num))
+        self.assertEqual(num, read_int(path))
+
+        num = 10.5
+        path = self._write_temp_file('read_int', content=str(num))
+        self.assertRaises(ValueError, read_int, path)
```

Comment:
Is this Cromwell's behavior, and is it the same for `10.0`?

We should probably implement what Cromwell does in this case, since if they implement this sort of duck typing for floats/ints would mean that "reasonable" workflows will run without error on Cromwell, but will error on Toil.  If not, ignore my comment.  If so, we should include a comment about why we diverge from the spec.

Reply:
Anything that is not an integer would not work, even `10.0`. Cromwell does a similar thing where it throws the following errors when passing `10.0` and `10.5` to `read_int()`: 

```
Caused by: common.exception.AggregatedMessageException: Error(s):
Could not evaluate expression: read_int(in_file): For input string: "10.0”
        at common.validation.Validation$ValidationTry$.toTry$extension1(Validation.scala:89)
…
```
```
Caused by: common.exception.AggregatedMessageException: Error(s):
Could not evaluate expression: read_int(in_file): For input string: "10.5”
        at common.validation.Validation$ValidationTry$.toTry$extension1(Validation.scala:89)
…
```

So, we should be good in these cases. This is a good point to bring up though, since I didn’t think about testing `10.0`.